plugins/browser-plugin-web-vitals/src/index.ts

```diff
@@ -0,0 +1,120 @@
+import { BrowserPlugin, BrowserTracker, addEventListener } from '@snowplow/browser-tracker-core';
+import { ReportCallback } from './types';
+import { WEB_VITALS_SCHEMA } from './schemata';
+import { dispatchToTrackersInCollection } from '@snowplow/browser-tracker-core';
+import { LOG, buildSelfDescribingEvent } from '@snowplow/tracker-core';
+
+const _trackers: Record<string, BrowserTracker> = {};
+
+declare global {
+  interface Window {
+    webVitals: {
+      onCLS(cb: ReportCallback): void;
+      onFID(cb: ReportCallback): void;
+      onLCP(cb: ReportCallback): void;
+      onINP(cb: ReportCallback): void;
+      onTTFB(cb: ReportCallback): void;
```

Comment:
Is the omission of `onFCP` deliberate?

Reply:
Yes, at the time it was deliberate, but I am now thinking it will not hurt that much adding it. And there could be some use cases for reporting it and observing it over time.